redis.c

```diff
@@ -730,8 +731,13 @@ redis_connect(INTERNAL_FUNCTION_PARAMETERS, int persistent)
         return FAILURE;
     }
 
+    /* Does the host look like a unix socket */
+    af_unix = (host_len > 0 && host[0] == '/') ||
+              (host_len > 6 && !strncasecmp(host, "unix://", sizeof("unix://") - 1)) ||
```

Comment:
`ZEND_STRL` may be used here and on the next line

Reply:
I actually used `ZEND_STRL` first, but it broke compilation on Windows since I guess `strncasecmp` is a macro there.

[Example failed run](https://github.com/phpredis/phpredis/actions/runs/3239297939/jobs/5308680621)

```
2022-10-13T03:04:57.6530144Z D:\a\phpredis\phpredis\redis.c(736): warning C4003: not enough arguments for function-like macro invocation 'strncasecmp'
2022-10-13T03:04:57.8142538Z D:\a\phpredis\phpredis\redis.c(736): error C2059: syntax error: ')'
```

Reply:
Windows is 🥴